keras/preprocessing/image.py

```diff
@@ -382,6 +391,7 @@ class ImageDataGenerator(object):
         featurewise_std_normalization: divide inputs by std of the dataset.
         samplewise_std_normalization: divide each input by its std.
         zca_whitening: apply ZCA whitening.
+        brightness: apply brightness
```

Comment:
What does this mean?

Reply:
@fchollet I guess I wasn't paying attention. I meant for it to mean, Use Random Brightness

Reply:
That's redundant with `brightness_range`. We use only one argument for such transforms.